<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*QToolButton{
		background-color: #44c767;
		border-radius: 4px;
		color: #ffffff;
		font-family: 'Roboto';
		margin-left:0.5em;
		font-size: 17px;
	}
QToolButton:hover{
			background-color:#8fe860;
	}

QToolBar{
	border-color:#8fe860;
}*/
/*QTextEdit{
		border-radius: 3px;
		border: 2px solid #00796b;
	}
QToolButton{
		background-color:#44c767;
		color:#ffffff;
		border-radius: 16px;
		margin-left : 1em;
		font-family:Arial;
		font-size:15px;
}
	
QToolButton:hover{
			background-color:#8fe860;
	}
*/
</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>458</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionN"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Abo&amp;ut</string>
    </property>
    <addaction name="actionInfo"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Optio&amp;ns</string>
    </property>
    <addaction name="actionLine"/>
    <addaction name="actionLexer"/>
    <addaction name="actionContinue"/>
    <addaction name="actionStop"/>
    <addaction name="actionClearTerm"/>
    <addaction name="actionClearEst"/>
    <addaction name="separator"/>
    <addaction name="actionRun"/>
    <addaction name="actionCancel"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionBreakpoints"/>
    <addaction name="actionTimes"/>
    <addaction name="actionTree"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
   <addaction name="menuTools"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="Barra">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset>
     <normalon>:/Open/Enter.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset>
     <normalon>:/Save/Download.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="icon">
    <iconset resource="Imagenes/resources.qrc">
     <normaloff>:/Info/Info.png</normaloff>:/Info/Info.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Info</string>
   </property>
  </action>
  <action name="actionLine">
   <property name="icon">
    <iconset>
     <normalon>:/Line/Pin.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Line</string>
   </property>
  </action>
  <action name="actionLexer">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Imagenes/resources.qrc">
     <normaloff>:/Lexer/Lightning.png</normaloff>:/Lexer/Lightning.png</iconset>
   </property>
   <property name="text">
    <string>L&amp;exer</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset>
     <normalon>:/Close/Cancel.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionContinue">
   <property name="icon">
    <iconset>
     <normalon>:/Continue/FlagBlue.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Continue</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset>
     <normalon>:/Stop/FlagYellow.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Stop</string>
   </property>
  </action>
  <action name="actionNew_2">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionN">
   <property name="icon">
    <iconset>
     <normalon>:/Add/Add.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="icon">
    <iconset>
     <normalon>:/Run/FlagGreen.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Run</string>
   </property>
  </action>
  <action name="actionCancel">
   <property name="icon">
    <iconset>
     <normalon>:/Cancel/FlagRed.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Ca&amp;ncel</string>
   </property>
  </action>
  <action name="actionClearEst">
   <property name="icon">
    <iconset>
     <normalon>:/Clear/Bin.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Clear Estado Var</string>
   </property>
  </action>
  <action name="actionClearTerm">
   <property name="icon">
    <iconset>
     <normalon>:/Clear/Bin.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Clear Terminal</string>
   </property>
  </action>
  <action name="actionBreakpoints">
   <property name="text">
    <string>Breakpoints</string>
   </property>
  </action>
  <action name="actionTree">
   <property name="text">
    <string>Tree</string>
   </property>
  </action>
  <action name="actionTimes">
   <property name="text">
    <string>Times</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Imagenes/resources.qrc"/>
 </resources>
 <connections/>
</ui>
